File: codechef/practice/FSQRT.java
import java.util.Scanner;

class FSQRT{
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int testcases = scan.nextInt();

        for(int i = 1;i<=testcases;i++){
            int number = scan.nextInt();
            double numRoot = Math.sqrt(number);
            int answer = (int)numRoot;
            System.out.println(answer);
        }
    }
}

File: codechef/practice/FSQRT.cpp
#include <iostream>
#include <math.h>

using namespace std;

int main(){
    int testcases;
    cin>>testcases;

    for(int i = 1;i<=testcases;i++){
        int number;
        cin>>number;
        double numRoot = sqrt(number);
        int result = (int)numRoot;
        cout << result << endl;
    }
    return 0;
}